packages/cli/README.md

```diff
@@ -441,6 +444,7 @@ ARGUMENTS
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
   -k, --key=key          DID Private Key
+  -s, --syncOption  Controls if the current stream state should be synced over the network or not. 'prefer-cache' will return the state from the node's local cache if present, and will sync from the network if the stream isn't in the cache. 'always-sync' always syncs from the network, even if there is cached state for the stream. 'never-sync' never syncs from the network.
```

Comment:
As per @stbrody :
> create doesn't need to expose a sync option, it should only be set on query-like commands, not write commands

Reply:
This part of the docs is for `tile:deterministic`, not for `tile:create`, @ukstv . I think the way changes were wrapped by GH was confusing when reviewing :).